soh/soh/z_message_OTR.cpp

```diff
@@ -153,4 +153,13 @@ extern "C" void OTRMessage_Init()
             "Tu as l'air de t'ennuyer. Tu veux&aller faire un tour?\x1B&%gOui&Non%w",
         }
     );
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_RANDO_SAVE_VERSION_WARNING,
+        {
+            TEXTBOX_TYPE_NONE_BOTTOM, TEXTBOX_POS_BOTTOM,
+            "This save was created&on a different build.&&Things may be broken.",
```

Comment:
not sure if "different build" will be clear enough for players, maybe "different version of SoH" (not sure if we have space for that on screen

Reply:
We have 4 lines that can be displayed "nicely", 5 can technically work but would render over the "file select window", and the width can be larger. I would say the maximum width is as long as it fits within a 4:3 ratio.

Let me try that out and share a screenshot.

Reply:
maybe
```
"This save was created on&a different version of SoH.&&Things may be broken."
```

Reply:
@briaguya-ai heres how this looks 

![image](https://user-images.githubusercontent.com/13861068/220789301-7f3d9113-532a-4eb7-bf18-9038e2f48478.png)

Reply:
i like it